src/tests/multihost/alltests/test_ldap_library_debug_level.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function
+import re
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from constants import ds_instance_name
+
+
+@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups')
+@pytest.mark.ldapliblevel
```

Comment:
Should this marker be added to pytest.ini file? I think `ldaplibdebuglevel` will be more clear